src/Pages/ContentListTable/components/StatusIcon.tsx

```diff
@@ -43,31 +77,50 @@ const StatusIcon = ({ status, error }: Props) => {
       );
     case 'Invalid':
       return (
-        <Tooltip position='top-start' content={error}>
           <Flex alignContent={{ default: 'alignContentCenter' }} direction={{ default: 'row' }}>
             <FlexItem spacer={{ default: 'spacerSm' }}>
               <ExclamationCircleIcon color={red} />
             </FlexItem>
             <FlexItem>
-              <StatusText color='red'>Invalid</StatusText>
+              <Popover
+                aria-label="invalid popover"
+                alertSeverityVariant='danger'
+                headerContent="Invalid"
+                headerIcon={<ExclamationCircleIcon/>}
+                bodyContent={popoverDescription(error, failedIntrospectionsCount, lastIntrospectionTime)}
+                position='left'
+                footerContent={popoverFooter(retryHandler, uuid)}
+              >
+                <Button variant="link" isInline>
+                    <StatusText color='red' isLink>Invalid</StatusText>
+                  </Button>
+             </Popover>
             </FlexItem>
           </Flex>
-        </Tooltip>
       );
     case 'Unavailable':
       return (
-        <Tooltip position='top-start' content={error}>
           <Flex alignContent={{ default: 'alignContentCenter' }} direction={{ default: 'row' }}>
             <FlexItem spacer={{ default: 'spacerSm' }}>
               <ExclamationTriangleIcon color={gold} />
             </FlexItem>
             <FlexItem>
-              <StatusText color='gold'>Unavailable</StatusText>
+              <Popover
+                aria-label="unavailable popover"
+                alertSeverityVariant='warning'
+                headerContent="Unavailable"
+                headerIcon={<ExclamationTriangleIcon/>}
+                bodyContent={popoverDescription(error, failedIntrospectionsCount, lastIntrospectionTime)}
+                position='left'
+                footerContent={popoverFooter(retryHandler, uuid)}
+              >
+                <StatusText color='gold' isLink>Unavailable</StatusText>
+              </Popover>
             </FlexItem>
           </Flex>
-        </Tooltip>
       );
     case 'Pending':
+    case '':
```

Comment:
After thinking about this, it may make more sense not to add this here. 
But rather add it to the check for the polling script within react-query. 
As say if we have a "" in the future with very few items and no "Pending" statuses. The update script would never run. 

That way we aren't masking the mistake from the API (or expecting it) and we are still refreshing the page every 10 seconds with the expectation that backend should updated the value.

Reply:
I did the first thing